sdk/samples/iot/README.md

```diff
@@ -138,6 +138,7 @@ To run the samples, ensure you have the following programs or tools installed on
     ```
 
   - For non-Linux based systems, download the [OpenSSL 1.1.1 LTS](https://www.openssl.org/source/openssl-1.1.1g.tar.gz) command line utility and follow the downloaded INSTALL document.
+- To use the certificate generation script (`generate_certificate.ps1`), you must have Powershell 7 installed. Please follow the installation instructions for your platform [here](https://github.com/PowerShell/PowerShell/tree/v7.0.3#get-powershell).
```

Comment:
Nit: What if PowerShell 8 comes out?  Presumambly this would keep working?  What happens 10 years off when PS7 is deprecated but we still link to it?  

Could we just have people go to latest PowerShell core or whatever right nomenclature is?

Reply:
I would reorder the sentence to match the others in this section:
"Have \[Powershell\](https://github.com/PowerShell/PowerShell/tree/v7.0.3#get-powershell) installed if running certificate samples.  This is required to run the certificate generation script `generate_certificate.ps1`. "